osu.Game.Tests/Visual/Editing/TestSceneEditorBeatmapCreation.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Testing;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Osu;
+using osu.Game.Screens.Edit.Setup;
+using osu.Game.Tests.Resources;
+using SharpCompress.Archives;
+using SharpCompress.Archives.Zip;
+
+namespace osu.Game.Tests.Visual.Editing
+{
+    public class TestSceneEditorBeatmapCreation : EditorTestScene
+    {
+        protected override Ruleset CreateEditorRuleset() => new OsuRuleset();
+
+        protected override bool EditorComponentsReady => Editor.ChildrenOfType<SetupScreen>().FirstOrDefault()?.IsLoaded == true;
+
+        public override void SetUpSteps()
+        {
+            AddStep("set dummy", () => Beatmap.Value = new DummyWorkingBeatmap(Audio, null));
+
+            base.SetUpSteps();
+        }
+
+        [Test]
+        public void TestCreateNewBeatmap()
+        {
+            AddStep("add random hitobject", () => EditorBeatmap.Metadata.Title = Guid.NewGuid().ToString());
```

Comment:
step name and action are two different things :wink: not sure which way you meant to go here

Reply:
My line of thought was "this test is failing from saving multiple non-unique beatmaps" so I was going to add a hitobject. Then realised changing the metadata was much simpler. Oops.